modules/cassandra/src/main/java/org/apache/ignite/cache/store/cassandra/utils/session/CassandraSessionImpl.java

```diff
@@ -248,8 +248,7 @@ else if (CassandraHelper.isPreparedStatementClusterError(e))
         if (assistant.processedCount() == 0)
             throw new IgniteException(errorMsg, error);
 
-        if (log != null)
-            log.warning(errorMsg, error);
+        log.warning(errorMsg, error);
```

Comment:
It will produce NullPointerException if Ignite was started with Basic configuration without Spring support added (see the same comment above)